wacz/main.py

```diff
@@ -163,6 +170,9 @@ def create_wacz(res):
     index_file = zipfile.ZipInfo("indexes/index.idx", now())
     index_file.compress_type = zipfile.ZIP_DEFLATED
 
+    pages_jsonl = zipfile.ZipInfo("pages/pages.jsonl", now())
```

Comment:
Slight nitpick, not sure if its better put these in the conditional since they're only used there. The other path opens them elsewhere, I think?